oeis.py

```diff
@@ -382,6 +382,32 @@ def A000108(start: int = 0, limit: int = 20) -> Collection[int]:
         sequence.append(int(r))
 
     return sequence
+def fibonacci(n):
+    if n < 0:
+        print("Incorrect input")
+    if n <= 1:
+        return 1
+    else:
+        return fibonacci(n - 1) + fibonacci(n - 2)
+
+
+def A000045(start, limit, plot=False):
+
+    numbers = []
+    y_label = []
+    for i in range(int(start), int(limit) + 1):
+        numbers.append(fibonacci(i))
+        y_label.append(i)
+
+    if plot:
+        with plt.style.context("dark_background"):
+            plt.plot(y_label, numbers, "r-o")
+
+        plt.xlabel("x label")
+        plt.ylabel("y label")
+        plt.title("A000045 - Fibonacci")
+        plt.show()
```

Comment:
This is now done by the main function, you don't have to put it in your function.